server/protocols/webadmin-integration-test/webadmin-integration-test-common/src/main/java/org/apache/james/webadmin/integration/WebAdminServerIntegrationTest.java

```diff
@@ -69,13 +68,10 @@
     protected static final String ALIAS_1 = "alias1@" + DOMAIN;
     protected static final String ALIAS_2 = "alias2@" + DOMAIN;
 
-    private GuiceJamesServer guiceJamesServer;
     private DataProbe dataProbe;
 
-    @Before
-    public void setUp() throws Exception {
-        guiceJamesServer = createJamesServer();
-        guiceJamesServer.start();
+    @BeforeEach
+    void setUp(GuiceJamesServer guiceJamesServer) throws Exception {
```

Comment:
why don't you make it a contract test (interface)?

Reply:
Mostly I don't want to grow the change set. I will invest that effort in another PR if you don't mine

Reply:
see #3042